src/main/java/ua/com/javarush/oleksandr/reddit/redditcloneabstract/dto/PostRequest.java

```diff
@@ -15,9 +11,7 @@
 @NoArgsConstructor
 @EqualsAndHashCode(of = "url")
 @Builder(builderMethodName = "with")
-public class PostRequestDTO {
-
-    private Long id;
```

Comment:
The 'id' here is intended for future functionality to perform updates on a post.

Reply:
Hmm, didn't think of that, I'll correct it